com.lettier.movie-monad.json

```diff
@@ -0,0 +1,1353 @@
+{
+  "app-id": "com.lettier.movie-monad",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.26",
+  "sdk": "org.gnome.Sdk",
+  "writable-sdk": true,
+  "command": "movie-monad",
+  "rename-desktop-file": "movie-monad.desktop",
+  "rename-icon": "movie-monad-icon",
+  "rename-appdata-file": "movie-monad.appdata.xml",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=wayland",
+    "--device=dri",
+    "--share=network",
+    "--socket=pulseaudio",
+    "--talk-name=org.gtk.vfs",
+    "--talk-name=org.gtk.vfs.*",
+    "--talk-name=org.gnome.SettingsDaemon",
+    "--talk-name=org.gnome.SettingsDaemon.MediaKeys",
+    "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
+    "--filesystem=host",
+    "--filesystem=xdg-run/dconf",
+    "--talk-name=ca.desrt.dconf"
+  ],
+  "build-options": {
+    "strip": false,
+    "no-debuginfo": true
```

Comment:
Why are you disabling these?

Reply:
> strip is false by default. 

Then remove it.

> no-debuginfo (enabled if strip is false) is disabled to keep the debug information in the ELF files. 

Correct. Don't do that. It just makes larger downloads for users.

Reply:
Again, it is the same in the [example](https://github.com/flathub/org.freedesktop.Bustle/blob/master/org.freedesktop.Bustle.json#L16) provided which is already in Flathub.

If removed, how does one use the extension to help debug the application out in the wild?

:+1:

Reply:
> If removed, how does one use the extension to help debug the application out in the wild?

They install `com.lettier.movie-monad.Debug`